runtimes/mlflow/tests/conftest.py

```diff
@@ -96,6 +96,15 @@ def model_settings(model_uri: str) -> ModelSettings:
     )
 
 
+@pytest.fixture
+def model_settings_scheme(model_uri: str) -> ModelSettings:
+    return ModelSettings(
+        name="mlflow-model",
+        implementation=MLflowRuntime,
+        parameters=ModelParameters(uri="file:" + model_uri),
```

Comment:
Rather than adding a new fixture, should we parametrise one of the tests to run with `file://` or no-schema. That way we add less boilerplate code to maintain later.

Reply:
I refactored the tests by pytest parameterization.